MyOnboarding/src/MyOnboardingApp.Contracts/Repository/IDatabaseConnection.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace MyOnboardingApp.Contracts.Repository
+{
+    public interface IDatabaseConnection
+    {
+        string GetDatabaseConnectionString { get; }
```

Comment:
```suggestion
        string DatabaseConnectionString { get; }
```
It is a property, don't name it like a method 👼